.eslintrc.yml

```diff
@@ -47,6 +48,26 @@ rules:
   func-style: ['error', 'declaration', { 'allowArrowFunctions': true }]
   new-cap: ['error', { 'capIsNew': true }]
   import/order: ['error', { 'newlines-between': 'never' }]
+  sort-class-members/sort-class-members:
+    [
+      'warn',
+      {
+        order:
+          [
+            'name',
+            'category',
+            'isDeprecated',
+            'route',
+            'examples',
+            'defaultBadgeData',
+            'render',
+            'constructor',
+            'fetch',
+            'transform',
+            'handle',
```

Comment:
I'm not sure about applying this order globally. It definitely makes sense to apply to `services/**/*.js` and the base classes, but it feels like applying this in dirs like `/lib` for example that don't contain service classes is a bit odd. For example I notice that running `lint` locally is throwing some warnings on files in `frontend` but its a coincidence that we're using the same method names.

Reply:
Yea, agreed that should be adjusted. Scoping it to `services` seems like the right thing to do.

Mostly we've dealt with that using separate eslintrcs, though I think there's also way to keep the config centralized and apply rules only to certain files.

Reply:
Yes you can do it all in one config file with glob patterns:

https://eslint.org/docs/user-guide/configuring#configuration-based-on-glob-patterns